### PR TITLE
Package ppx_matches.0.1

### PR DESCRIPTION
**Pull request by wrbs**
### `ppx_matches.0.1`
Small ppx to help check if a value matches a pattern
Translates [%matches? <PATTERN>] into (function | <PATTERN> -> true | _ -> false)

---
* Homepage: https://github.com/wrbx/ppx_matches
* Source repo: git://github.com/wrbs/ppx_matches
* Bug tracker: https://github.com/wrbx/ppx_matches/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/ppx_matches/ppx_matches.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+description: """
+Translates [%matches? <PATTERN>] into (function | <PATTERN> -> true | _ -> false)
+"""
+maintainer: "Will Robson <will at will robson dot uk>"
+authors: "Will Robson <will at will robson dot uk>"
+license: "MIT"
+homepage: "https://github.com/wrbx/ppx_matches"
+bug-reports: "https://github.com/wrbx/ppx_matches/issues"
+dev-repo: "git://github.com/wrbs/ppx_matches"
+depends: [ "ocaml" "ppxlib" "dune" ]
```
Comment (kit-ty-kate) [inline packages/ppx_matches/ppx_matches.0.1/opam:12]:
```suggestion
depends: [
  "ocaml" {>= "4.04"}
  "ppxlib" {>= "0.15.0"}
  "dune" {>= "2.9"}
]
```
I'm not sure what are your minimal requirements for ppxlib and OCaml but this is most likely the bare minimum.
For dune, your own `dune-project` file says you require at least 2.9.

**Comment by mseri** [comment]
Thanks

**Comment by mseri** [comment]
Feel free to announce the release on the discuss.ocaml.org forum. We have a community category and an announcement tag for that purpose

**Comment by kit-ty-kate** [comment]
@wrbs could you return the fixes in the opam file upstream so that they cane be taken into account next time? Also some of the URLs need to be fixed (homepage and bug-tracker) `wrbx --> wrbs`

**Pull request merged**